Suko/API/SUKAPIManager.m

```diff
@@ -54,11 +53,12 @@ - (void)fetchAnimeWithID:(NSNumber *) malID completion:(void(^)(SUKAnime* anime,
         completion(animeObj, nil);
     } failure:^(NSURLSessionTask *operation, NSError *error) {
         NSLog(@"Error: %@", error);
```

Comment:
Small nit, maybe add some more context in the error logging so that when we get error we know which endpoint is throwing the error. e.g. "Error fetching anime with ID: %@"